toc.json

```diff
@@ -96,7 +105,7 @@
     "key": "simcore/services/dynamic/jupyter-neuron",
     "name": "jupyter-neuron",
     "type": "dynamic",
-    "version": "2.13.0"
+    "version": "1.1.0"
   },
```

Comment:
is this correct?

Reply:
Looking at the folder and repo history, 1.1.0 is the correct value. Not sure how the other version happen